src/account/UpgradeableModularAccount.sol

```diff
@@ -109,21 +109,21 @@ contract UpgradeableModularAccount is
 
     /// @notice Initializes the account with a set of plugins
     /// @param plugins The plugins to install
-    /// @param manifestHashes The manifest hashes of the plugins to install
+    /// @param manifests The manifests of the plugins to install
     /// @param pluginInstallDatas The plugin install datas of the plugins to install
     function initialize(
         address[] memory plugins,
-        bytes32[] memory manifestHashes,
+        PluginManifest[] calldata manifests,
```

Comment:
Minor nitpick, but I think we could rename it to a simpler word than manifest now-- it's no longer a request from the plugin, so maybe something like `pluginInstallConfig`?

Reply:
I like this idea. But tbh I thought manifest ~= install config? Thought I guess manifest has connotations of also holding other metadata.

Reply:
Yeah definitely a super-small thing, I think it's just a less frequently used word that a bunch of people might not really understand at first (maybe relevant for non-native speakers, I guess?) although it's fairly straightforward if you look at the data structure.